stack-8.2.1.yaml

```diff
@@ -40,6 +40,8 @@ extra-deps:
 - sorted-list-0.2.1.0
 - syz-0.2.0.0
 - yi-rope-0.11
+- github: Bubba/haskell-lsp-test
```

Comment:
Please use a submodule, like all the other dependencies.  It makes life a lot easier for those of us with constrained bandwidth at times.